src/components/SidebarItems.tsx

```diff
@@ -48,28 +48,45 @@ const SidebarItems: FC<{ cat: union[number]; active: boolean }> = ({
   const router = useRouter();
   const route = routeMap[cat];
 
+  const [mouseOver, setMouseOver] = useState(0);
```

Comment:
I would recommend using a boolean here for clarity

Reply:
Just made this change, should be finished.